### ISSUE TITLE
Add default options for a11y-friendly (VI) connections

### ISSUE THREAD

**Issue by DavidRieman** [DavidRieman/WheelMUD#184]
It would be nice if we can determine a desire for screen reader friendly output even before the splash screen is sent, so we can have the screen reader printer output appropriately reduced output like a welcome message in place of any ASCII art, etc.

One avenue to investigate may be whether there is wide-spread client support for MTTS (https://tintin.mudhalla.net/protocols/mtts/) but this probably doesn't exist in standard Telnet clients that may exist in the OS, or other legacy telnet terminals and such for users going for a really _nostalgic_ sort of approach.  It would probably be nice to have other way(s) to control this (too).

Another option may be accepting that users with screen reader on will get garbage read-outs that they have to skip, until they issue some command on the login screen that switches them to VI-friendly mode.  This isn't perfect as a work-around but may be a "budget" work-around solution.

Another option is to configure our Telnet stack to open a second port for listening to incoming connections specifically for accessibility (a11y) read-out users (such as visually impaired users).  This could be a pre-existing option configured through the App.config.  If enabled, then connections to this port get flagged with some property designating the user should not receive ASCII graphics (nor real graphics) and our default splash screen system would customize appropriately.  This route depends on #183.  Perhaps this is the preferable option so far...
